nameko/web/server.py

```diff
@@ -47,10 +64,8 @@ def __init__(self):
 
     @property
     def bind_addr(self):
-        return (
-            self.container.config.get(WEB_SERVER_HOST_CONFIG_KEY, ''),
-            self.container.config.get(WEB_SERVER_PORT_CONFIG_KEY, 8000),
-        )
+        address_str = self.container.config.get(WEB_SERVER_CONFIG_KEY, '8000')
```

Comment:
Eventlet will bind to ``0.0.0.0`` if you pass an empty string for the address, so ``0.0.0.0:8000`` would be explicit.

``` python
>>> sock = eventlet.listen(('', 8000))
>>> sock.getsockname()
... ('0.0.0.0', 8000)
```